client/js/_author/components/assessments/question_types/audio_upload.jsx

```diff
@@ -20,6 +21,24 @@ export default class AudioUpload extends React.Component {
     return seconds;
   }
 
+  static rangeWarning() {
+    return (
+      <span className="author--c-inline-error">Please enter a positive number under 240</span>
+    );
+  }
+
+  constructor(props) {
+    super(props);
+    const timeLimit = AudioUpload.getAudioLimit(props.item)
+      || `${AudioUpload.MAX_TIME}`;
+    const displayWarning = timeLimit > AudioUpload.MAX_TIME;
```

Comment:
It is a little bit tricky because qbank sends it as a string, but only accepts it as an int. This is not right though. Thanks for noticing.